src/api/streams/index.ts

```diff
@@ -0,0 +1,172 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
I think this should be a last resort, not the default, like it now seems to be. Could we put a bit more effort in typings so that we don't need to do this? This carries also forward: it is way too easy to use `any` also in the future in these files.